File: app/src/main/java/com/starostinvlad/fan/NewsScreen/NewsFragmentContract.java
package com.starostinvlad.fan.NewsScreen;

import com.starostinvlad.fan.BaseMVP.MvpView;
import com.starostinvlad.fan.GsonModels.News;

import java.util.List;

public interface NewsFragmentContract extends MvpView {
    void fillListView(List<News> newsList);
    void showLoading(boolean show);

    void refreshListView();
    void alarm(String message);
}

File: app/src/main/java/com/starostinvlad/fan/NewsScreen/NewsFragmentContract.kt
package com.starostinvlad.fan.NewsScreen

import com.starostinvlad.fan.BaseMVP.MvpView
import com.starostinvlad.fan.GsonModels.News
import kotlin.Throws

interface NewsFragmentContract : MvpView {
    fun fillListView(newsList: List<News>)
    fun showLoading(show: Boolean)
    fun refreshListView()
    fun alarm(message: String?)
}